packages/bot/src/utils/firestore.js

```diff
@@ -77,17 +76,17 @@ const firestore = (() => {
         }
     };
 
-    const onStopBot = (login_id) => {
+    const onStopBot = (login_id, server_time) => {
```

Comment:
Why pass `server_time` into the function? :thinking: isn't it available on the outer scope?

Reply:
@reza-fs i have to get server_time from `core.common.server_time`. didn't understand what you mean?

Reply:
@reza-fs i tried to make it nice but firestore is a helper class so i have to pass store in.